src/ffi/w3c/cred_offer.rs

```diff
@@ -0,0 +1,25 @@
+use crate::ffi::cred_offer::anoncreds_create_credential_offer;
+use crate::ffi::error::ErrorCode;
+use crate::ffi::object::ObjectHandle;
+use ffi_support::FfiStr;
+
+/// Create Credential Offer according to the AnonCreds specification
+/// Note that Credential Offer still will be legacy styled (the same as result of anoncreds_create_credential_offer)
+///
+/// # Params
+/// schema_id:              id of schema future credential refers to
+/// cred_def_id:            id of credential definition future credential refers to
+/// key_proof:              object handle pointing to credential definition key correctness proof
+/// cred_offer_p:           reference that will contain created credential offer (in legacy form) instance pointer
+///
+/// # Returns
+/// Error code
+#[no_mangle]
+pub extern "C" fn anoncreds_create_w3c_credential_offer(
```

Comment:
I likely have missed some previous discussions surrounding this, but why is this method added? I have no real objection to it, just more curious.

Reply:
We discussed during one of the community group calls. 
There are 6 methods for running issuance/verification flow methods in total. 
We have to duplicate at least 4 of them: `create_credential`, `process_credential`, `create_presentation`, `verify_presentation`. 

`credential_offer` and `credential_request` are not changed for the current moment. It's clearer to use the same set of methods for running the whole `w3c` credential issuance process. Also, potentially offer and request can be adjusted for W3C standard somehow in future.